src/main/resources/org/jenkinsci/plugins/workflow/cps/steps/ParallelStep/help.html

```diff
@@ -1,14 +1,69 @@
 <div>
     <p>
-        Takes a map from branch names to closures and an optional argument <code>failFast</code>
-        which will terminate all branches upon a failure in any other branch:
-    </p>
-    <pre>
-    parallel firstBranch: {
-        // do something
-    }, secondBranch: {
-        // do something else
-    },
-    failFast: true|false
-    </pre>
+      Takes a map from branch names to closures and runs the closures in
+      parallel. The optional <code>failFast</code> option terminates
+      all jobs as soon as any job fails.
+    </p>
+    <p>
+      Synopsis:
+      <pre>
+        parallel firstBranch: {
+            // do something
+          }, secondBranch: {
+            // do something else
+          },
+          failFast: true|false
```

Comment:
```suggestion
          failFast: true
```

to make this a syntactically legal example.